packages/blockchain-wallet-v4-frontend/src/data/balances/README.md

```diff
@@ -0,0 +1,39 @@
+# Wallet Balances Selectors
+
+## Overview
+Almost every page and feature in the Wallet relies on getting balances in some capacity. 
+There are many variations on what type of balance data you may need. 
+This document will outline each of the public selectors available to help you decide which selectors you should be using.
+
+## Balance Selector Definitions
+All the selectors outlined below are available on the `data.selectors.balances` namespace.
+
+### Total Balances
+These selectors should be used if you need the total balance of the entire wallet or of a specific coin.
+Total balance means custodial (Trading, Buy/Sell, Interest, etc.) + non-custodial + fiat (if applicable).
+
+- `getCoinTotalBalance` - given `CoinType` or `FiatType`, will return its total balance across all accounts
+- `getCoinBalancesTypeSeperated` given `CoinType`, will return its balances in a list seperated by type ([non-custodial, custodial])
```

Comment:
small spelling error (separated)

Reply:
fixed spelling error. added a note on this selector so it will hopefully not be abused